### PR TITLE
JavaScript Capstone Project

### PR DESCRIPTION
**Pull request by Tufahel**
# Description 
> This is a web app where users can view all pokemon with their characteristics, abilities, and types. Data is fetched from the Pokeapi. Here users can like, comment, and view like and comments of him/her and others.

# General requirements

- [ ] There are no linter errors.
- [ ] Followed the Github flow.
- [ ] Followed the Gitflow.
- [ ] Used Webpack.
- [ ] The documentation is professional.
- [ ] HTML/CSS and JavaScript requirements
- [ ] Followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- [ ] Followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

# Milestones:

- [ ]  Milestone 1: Project Setup and topic.
- [ ]  Milestone 2: Creation of the Home interface. Creation of the comment and the reservation pop-ups
- [ ]  Milestone 3: Creation of the communication interfaces with the APIs
- [ ]  Milestone 4: Implementation of the likes, comments, and reservations functionalities
- [ ]  Milestone 5: Writing tests to ensure the robustness of the application

# Additional Requirement:

> We apologize for the submission delays. It was due to some difficulties handling the API.

> Here is Live [Link](https://tufahel.github.io/second-capstone/)

> Here is the video [Link](https://drive.google.com/file/d/1YU3rETpxCsQ0tuVQt4YCIk9aXmPcik6a/view)

### CODE REVIEW

File: src/modules/displayPop.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+export default async function displayPop(data, comments) {
+  const moves = data.moves['0'].move.name;
+  const { weight } = data;
+  const { height } = data;
+  const ability = data.abilities[0].ability.name;
+  const { name } = data;
+  const { id } = data;
+  const img = data.sprites.other['official-artwork'].front_default;
+  const popUp = document.querySelector('.pop-up');
+  // const comment = await getpostInvolveCommentApi(id);
```
Comment (Deepakdanger) [inline src/modules/displayPop.js:12]: - [ ] I would recommend removing the commented code to make the codebase cleaner.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,45 @@
...
 
 ## Getting Started
 To get a local copy up and running you just need to follow the following steps;
 - Clone this repository with
-git clone https://github.com/Tufahel using your terminal or command line.
+git clone https://github.com/Tufahel/second-capstone.git using your terminal or command line.
 - run code . to open it in vscode
+- run `npm install` in terminal
+- run `npm start`
+Now you are ready to use this web app.
```
Comment (Deepakdanger) [inline README.md:28]: - [ ] In the Readme file Please add the command to test the project as well. [ Hint: `npm test`]

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Tufahel & @Banstein, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/modules/displayPop.js (hunk 1 above)
Comment (Tufahel) [reply src/modules/displayPop.js]: Thank you for your review. We have changed it.

File: README.md (hunk 2 above)
Comment (Tufahel) [reply README.md]: Thank you for your review, we have updated it.

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi Team🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**